kernel/kernel/config-arm64

```diff
@@ -392,7 +397,7 @@ CONFIG_ARCH_ENABLE_SPLIT_PMD_PTLOCK=y
 CONFIG_SECCOMP=y
 CONFIG_PARAVIRT=y
 # CONFIG_PARAVIRT_TIME_ACCOUNTING is not set
-# CONFIG_KEXEC is not set
+CONFIG_KEXEC=y
```

Comment:
do we really want to enable it?

Reply:
No, I got a list of everything added and removed when I ran `make olddefconfig`. I plan on going through that today and fixing up things we don't want to enable.